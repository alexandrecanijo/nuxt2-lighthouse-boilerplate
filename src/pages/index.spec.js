import { shallowMount, createLocalVue } from '@vue/test-utils';
import IndexPage from './index.vue';

describe('Index Page', () => {
  const localVue = createLocalVue();

  const input = {
    localVue,
    stubs: {
      Tutorial: true,
    },
  };

  it('test snapshot', () => {
    const wrapper = shallowMount(IndexPage, input);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
