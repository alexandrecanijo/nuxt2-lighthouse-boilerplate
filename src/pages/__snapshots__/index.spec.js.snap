// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Page test snapshot 1`] = `"<tutorial-stub></tutorial-stub>"`;
